watir-rspec.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = Watir::RSpec::VERSION
 
-  gem.add_dependency "rspec", "~>2.0"
+  gem.add_dependency "rspec", "~>3.0"
```

Comment:
See https://github.com/rspec/rspec-core/blob/master/lib/rspec/core/formatters.rb 
The register code is  @ line 81

If you don't register you get the warning on deprecated formatter and the formatteri isn't used : see line 149

There isn't much documentation on this, but as far as I can see there are 2 options:
* register your formatter 
* use the rspec-legacy_formatter gem & the original formatter (you have to rename the before_all_ivars method to before_context_ivars;  see commit https://github.com/rspec/rspec-core/commit/639db76e5fd0e845ae6c44527a0ba750cb934acd  example_group.rb file)